src/managing-workflow/production-deployments.md

```diff
@@ -46,6 +46,12 @@ Please see the following documentation: [Customizing Controller][]
 Using TLS to encrypt traffic (including Workflow client traffic, such as login credentials) is
 crucial. See [Platform SSL][] for the platform.
 
+## Scale Routers
+
+If all router pods in your cluster become unavailable then you will be unable to access the workflow API or
+any deployed applications. To reduce the potential of this happening it is recommended that you scale the
+deis-router ReplicaController to run more than one replica controller. This can be accomplished by running
```

Comment:
FYI you're not scaling replica controllers. The replica controller manages pods, so it'd be better to say

>  To reduce the potential of this happening, it is recommended that you scale the Router's ReplicaController to run more than one pod

Though I agree with keerthan on how we're only recommending one replica for v2 at this time.